docker-compose.yml

```diff
@@ -521,7 +521,7 @@ services:
         - SA_PASSWORD=${MSSQL_PASSWORD}
         - ACCEPT_EULA=Y
       volumes:
-        - mssql:/var/opt/mssql
+        - ${DATA_PATH_HOST}/mssql:/var/lib/mysql
```

Comment:
otherwise there will be an error
>ERROR: Named volume "mssql:/var/opt/mssql:rw" is used in service "mssql" but no declaration was found in the volumes section.

during `docker-compose up`